modules/katamari/src/main/ts/ephox/katamari/api/Arr.ts

```diff
@@ -182,7 +182,7 @@ export const findUntil: {
 
 export const find: {
   <T, U extends T>(xs: ArrayLike<T>, pred: ArrayGuardPredicate<T, U>): Optional<U>;
-  <T = any>(xs: ArrayLike<T>, pred: ArrayPredicate<T>): Optional<T>;
+  <T>(xs: ArrayLike<T>, pred: ArrayPredicate<T>): Optional<T>;
```

Comment:
This is a breaking change to the types which could cause a number of things to fail to compile (which is probably a good thing). It however doesn't seem like it's needed for this PR, so if you want to make this change, please log a separate PR.

- [ ] Break this out into a separate PR if needed, including a changelog entry to state it no longer defaults to any.

Reply:
Reverted